posts/2023-07-25-23.0.0.7.adoc

```diff
@@ -0,0 +1,157 @@
+
+To start using the hosted Javadoc, navigate to the REFERENCE section of the link:https://openliberty.io/docs/latest/overview.html[docs]. 
+
+[#bugs]
+== Notable bugs fixed in this release
+
+
+We’ve spent some time fixing bugs. The following sections describe just some of the issues resolved in this release. If you’re interested, here’s the  link:https://github.com/OpenLiberty/open-liberty/issues?q=label%3Arelease%3A23007+label%3A%22release+bug%22[full list of bugs fixed in 23.0.0.7].
+
+* link:https://github.com/OpenLiberty/open-liberty/issues/25368[GlobalOpenTelemetry is missing public methods]
++
+Liberty overrides the `GlobalOpenTelemetry` class to return a no-op.  Liberty does not allow users to set the class and, instead, CDI injection should be used to acquire an `OpenTelemetry` object.
++
+However the override is missing public methods, which causes an error to occur if called.  For example:
++
+[source]
+----
+Caused by: java.lang.NoSuchMethodError: io/opentelemetry/api/GlobalOpenTelemetry.getPropagators()Lio/opentelemetry/context/propagation/ContextPropagators; (loaded from file:/home/gb110303/workspaces/open-liberty/dev/build.image/wlp/lib/io.openliberty.io.opentelemetry_1.0.77.jar by org.eclipse.osgi.internal.loader.EquinoxClassLoader@7be84dfc[io.openliberty.io.opentelemetry:1.0.77.202305162152(id=156)]) called from class io.opentelemetry.opentracingshim.OpenTracingPropagatorsBuilder (loaded from file:/home/gb110303/workspaces/open-liberty/dev/build.image/wlp/usr/servers/Telemetry10Shim/workarea/org.eclipse.osgi/52/data/cache/com.ibm.ws.app.manager_0/.cache/WEB-INF/lib/opentelemetry-opentracing-shim-1.19.0-alpha.jar by com.ibm.ws.classloading.internal.AppClassLoader@76e93c4a).
+at OpenTracingPropagatorsBuilder.<init>(OpenTracingPropagatorsBuilder.java:16)
+	at OpenTracingPropagators.builder(OpenTracingPropagators.java:32)
+	at OpenTracingShim.createTracerShim(OpenTracingShim.java:61)
+	at OpenTracingShimServlet.createShim(OpenTracingShimServlet.java:46)
+	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
+	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
+	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
+----
++
+This issue now resolved and calling a method on `GlobalOpenTelemetry` delegates to the no-op `OpenTelemetry` class.
+
+* link:https://github.com/OpenLiberty/open-liberty/issues/25479[Unable to make field long java.nio.Buffer.address accessible when using Java 17]
++
+During server start with `com.ibm.ws.tcpchannel.useZosAio=true` specified, an FFDC is created when running with Java 17 on z/OS:
```

Comment:
do we post z/Os bugs on Open Liberty? My understanding was OL technically doesn't support running on z/Os